openfisca_aotearoa/variables/live_organ_donor_compensation.py

```diff
@@ -83,7 +83,7 @@ class sum_of_earnings_in_last_52_weeks(Variable):
     value_type = float
     entity = Person
     label = u"Total earnings over last 52 weeks"
-    reference = "http://organiccoders.allengeer.com/brk-1234-abcdfcrg234356/"
+    reference = "https://github.com/OrganicCoders/openfisca-aotearoa/blob/master/openfisca_aotearoa/definitions/earnings_as_an_employee.md"
```

Comment:
This is not a stable URL: if the file gets renamed it would yield a 404. You can obtain a stable URL by navigating to this URL and pressing `y` on your keyboard 🙂